src/main/java/constants/skills/Bowmaster.java

```diff
@@ -26,7 +26,9 @@
  */
 public class Bowmaster {
     public static final int MAPLE_WARRIOR = 3121000;
+    public static final int VENGEANCE = 3121001;
```

Comment:
"Vengeance" is no real bowmaster skill

Reply:
It does show up on BBB but the background is a light red - wonder if it's a dummy skill that isn't actually used.  Removed anyway.